src/reducers/ui-state-updaters.js

```diff
@@ -103,9 +106,10 @@ export const DEFAULT_MAP_CONTROLS = [
     ...final,
     [current]: DEFAULT_MAP_CONTROLS_FEATURES
   }),
-  {}
+  {mapLocale: {show: false, active: false, activeMapIndex: 0}}
```

Comment:
where is this udes?

Reply:
At the moment this is used only during development and I will use it in the application based on Kepler. I thought that it would be nice to include this also in core, but since it is not (probably) in line with design of other components, I decided to leave it hidden by default.

Reply:
At the moment this is used only during development and I will use it in the application based on Kepler. I thought that it would be nice to include this also in core, but since it is not (probably) in line with design of other components, I decided to leave it hidden by default.